lms/djangoapps/courseware/tests/test_video_handlers.py

```diff
@@ -188,14 +188,23 @@ def test_handle_ajax_for_speed_with_nan(self):
         self.assertEqual(self.item_descriptor.speed, 1.0)
         self.assertEqual(self.item_descriptor.global_speed, 1.0)
 
-    def test_handle_ajax(self):
 
+    def test_handle_ajax_for_is_complete_video(self):
+        self.item_descriptor.handle_ajax('save_user_state', {'is_complete': True})
+        self.assertEqual(self.item_descriptor.is_complete, True)
+
+        # verify that the is_complete is still true
```

Comment:
I don't think we need this part of test. If we want to test both `True/False` values being set we should make this a data driven test.